README.md

```diff
@@ -60,18 +60,22 @@ To ensure a successful deployment, you will want to ensure that:
    to actually install RPM or DEB packages on the nodes. If you do not have this
    capability and you want to protect the nodes, you have to install using a
    privileged container.
-1. CrowdStrike's Helm Operator is a project, not a product, and released to the
-   community as a way to automate sensor deployment to kubernetes clusters. The
-   upstream repository for this project is
-   [https://github.com/CrowdStrike/falcon-helm](https://github.com/CrowdStrike/falcon-helm).
-1. The Helm Operator is IN DEVELOPMENT AND NOT PRODUCTION READY, so do not use
-   it! Use Helm Charts instead.
 
-# Installation
+# Quick Note Regarding Support
+🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 
```

Comment:
I feel bit uneasy about the ration of fireplaces to the fire fighter trucks. I wish there was more fire fighter trucks. Just to err on the side of caution.

Reply:
Updated!